openlibrary/templates/account/password/reset.html

```diff
@@ -3,7 +3,7 @@
 $var title: $_("Reset Password")
 
 <div id="contentHead">
-    <h1>$_("Reset Your Password")</h1>
+    <h1>$_("Reset Password")</h1>
```

Comment:
ok :)

Reply:
There were, and still probably are, many near matches. I tried to consolidate them into one, but if there are preferred ways to phrase things let me know and I'll change it.

I was going for consolidation of messages rather than editing the content here! Hence the WIP :)